data/ships.txt

```diff
@@ -203,7 +203,8 @@ ship "Bactrian"
 		"Ramscoop"
 		
 		"X4700 Ion Thruster"
-		"X5200 Ion Steering"
+		"X1700 Ion Thruster"
+		"X4200 Ion Steering"
```

Comment:
I think I would be better to have space left over to avoid stacking the engines.